app/src/main/java/com/veniosg/dir/mvvm/model/FileHolder.java

```diff
@@ -140,6 +144,28 @@ public String getFormattedSize(Context c, boolean recursive){
 		return Formatter.formatFileSize(c, getSizeInBytes(recursive));
 	}
 
+	public String getNumberOfItems(Context context) {
+		if (context == null) {
+			return EMPTY_STRING;
+		}
+
+		boolean isDirectory = mFile.isDirectory();
+
+		if (!isDirectory) {
+			return EMPTY_STRING;
+		}
+
+		int numOfDirectoryItems = mFile.listFiles().length;
+
+		if (numOfDirectoryItems == 0) {
```

Comment:
This should all be handled by a single plural. This will also force a single point of return for the happy case.

Reply:
@veniosg Plurals in English are a hard thing. Zero is not treated as a special case in English especially. Check this for example https://stackoverflow.com/questions/5651902/android-plurals-treatment-of-zero! Maybe I can check the current configuration or locale and only do it in English.

Reply:
Oh of course, good catch! In that case, it's acceptable to keep `0 items` in the zero case in English to let the system handle plural localisation. This would involve duplicating the value of the many_items resource, but OneSky is smart enough to handle that so don't worry.